mkctr.go

```diff
@@ -243,25 +276,30 @@ func fetchAndBuild(bp *buildParams) error {
 		if err != nil {
 			return err
 		}
-		if !bp.publish {
-			logf("not pushing")
-			return nil
-		}
 
-		img = mutate.Annotations(img, bp.annotations).(v1.Image) // OCI annotations
+		switch {
+		case bp.publish:
```

Comment:
This new `--out` flag seems to be mutually exclusive with `--push`. Should we show an error message if both of them are specified?